src/components/picker/picker.tsx

```diff
@@ -98,6 +97,9 @@ export class Picker {
 
     /**
      * True if the dropdown list should be displayed without cutting the content
+     *
+     * @deprecated This functiallity dosen't work anymore and should be
+     * handled by the searcher insted
```

Comment:
Small update on this. The picker wasn't actually limiting the number of items in the list, and so this property wasn't to avoid that. Instead, this property was there to allow the dropdown to become high enough to show all items in the list, regardless of whether that made the dropdown extend outside the screen. Thus, if not all items fit on screen, the user would scroll the whole page instead of the content of the dropdown.

It seems this hasn't worked lately though. Compare the _With icons and displaying full list without cutting content_ example of version [30.0.1](http://lime-documentation-lime-elements.s3-website-eu-west-1.amazonaws.com/versions/30.0.1/#/component/limel-picker/) (working) and version [30.0.2](http://lime-documentation-lime-elements.s3-website-eu-west-1.amazonaws.com/versions/30.0.2/#/component/limel-picker/) (broken). Type an `a` in the input to get a long list of results.

As we found previously, this property was originally added in commit 7f0a31a9e51f76acb866058d7aadde43771e0cd7, but I finally managed to find the PR for that commit: #393, fixing issue #363.

We should test the stuff listed in that issue, to make sure we don't reintroduce any of those problems.

Reply:
> to make sure we don't reintroduce any of those problems

(I don't think we will, to be fair. There's a lot of things that are different about the dropdown now: we calculate the size, we can pop it either above or below the trigger, you can scroll the page while the dropdown is visible, etc.)

Reply:
Oh, yeah, what I was _going_ to say was that we should update this comment a bit. Perhaps something like

```suggestion
     * @deprecated This was used for a workaround, and isn't needed any
     * longer. Setting it has no effect, and the property will be removed
     * in the next major version.
```

Reply:
nice that you found the issues! 🎉  For me both of the versions are broking, I don't know if I do anything wrong when I tested them. 

"If not all items fit on screen, the user would scroll the whole page instead of the content of the dropdown."
I don't see why we want to be able to switch between scrolling in the whole page and scrolling in the dropdown menu. 🤔

Reply:
Yes I agree and will update it 🙂

Reply:
> nice that you found the issues! 🎉 For me both of the versions are broking, I don't know if I do anything wrong when I tested them.

Hmm… I might have linked the wrong versions or something. I'll take a look and post a video showing what I mean.

> "If not all items fit on screen, the user would scroll the whole page instead of the content of the dropdown."
> I don't see why we want to be able to switch between scrolling in the whole page and scrolling in the dropdown menu. 🤔

I think back when that issue was created, we couldn't pop the dropdown _above_ the trigger, so we always just had the space _below_ the trigger to work with. That meant that if the trigger was placed at the bottom of the page, the dropdown would become tiny, sometimes smaller than a single list-item in height. Coupled with the fact that back then, when a dropdown was visible, the page behind it couldn't be scrolled, and we couldn't place the dropdown above the item instead of below it, this led to some workarounds that perhaps doesn't seem very obvious today 😄

Today though, all of those compounding problems have been fixed, and we should never end up in the situation that the dropdown is opened downward and shrunk to less than a single item in height, because we'll just be popping the dropdown upwards instead 👍

Reply:
The first example shown is v30.0.1 where the dropdown becomes longer than the page (thus what the prop is supposed to do), and the second example is v30.0.2 where the dropdown is shorter and gets a scrollbar.

https://user-images.githubusercontent.com/154725/106133121-1f522380-6165-11eb-84b5-dda4173101e2.mov

To clarify, we should still remove the behaviour in the first example 🙂